### PR TITLE
fix(html): perf and fixes

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Here:
- perf improvement:
```
Gnuplot not found, using plotters backend
Benchmarking html/minify/document/css_spec
Benchmarking html/minify/document/css_spec: Warming up for 3.0000 s
Benchmarking html/minify/document/css_spec: Collecting 10 samples in estimated 5.4698 s (110 iterations)
Benchmarking html/minify/document/css_spec: Analyzing
html/minify/document/css_spec
                        time:   [48.982 ms 49.110 ms 49.259 ms]
                        change: [-12.105% -6.2772% -2.4870%] (p = 0.03 < 0.05)
                        Performance has improved.
```
- bugfix for `EqIgnoreSpan` due `raw`

**BREAKING CHANGE:**

Yes, no we use `JsWord` for utils

**Related issue (if exists):**

No

### CODE REVIEW

**Comment by alexander-akait** [comment]
@kdy1 Rebased anf fixed

File: crates/swc_html_minifier/benches/full.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub fn bench_files_document(c: &mut Criterion) {
     );
     bench_file(
         "github",
-        Path::new("../swc_html_parser/benches/files/github_com_17_05_2022.html"),
+        Path::new("../swc_html_parser/benches/files/github_com_17_05_2022. html"),
```
Comment (kdy1) [inline crates/swc_html_minifier/benches/full.rs:38]: Seems like a mistake?

Comment (alexander-akait) [reply crates/swc_html_minifier/benches/full.rs]: @kdy1 Yeah :smile: Fixed

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_html_ast

**Pull request merged**